JEAN/PaginaWebJean/Contacto.html

```diff
@@ -0,0 +1,26 @@
+<html>
+<head> 
+    
+    <h1>Programador Web</h1>
+    <h2>Jean  Mungarrieta</h2>
+    <img src="/IMAGENES/pp.jpg" alt="">
+</head>
+<body>
+    <p>Soy Jean Cralos Mungarrieta tengo 31 años, naci en venezuela y actualmente tengo ya 4 años viviendo en España </p>
+     <p>Mis peliculas favoritas son</p>
+    
+    <li>Volver Al Futuro</li>
+    <li>Forrest Gump</li>
+    <li>Space Jam</li>
+   
+    <P>What is Lorem Ipsum?
+        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+        
+        Why do we use it?
+        It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).</P>
+   
+   <p>este es mi papa</p>
+        <video width="320" height="240" controls><source src="/video/ElHuevoEmbarazado.mp4"></video>
+        
+
```

Comment:
missing body and html close tags. modern browser will render this html page, but older not